franklin/builder/models.py

```diff
@@ -60,38 +63,20 @@ def is_deployable_event(self, github_event):
 
     def build(self):
         url = os.environ['BUILDER_URL']
-        headers = {'content-type': 'application/json'}
-        body = {
-                    "github_token": self.oauth_token,
-                    "git_hash": self.git_hash,
-                    "repo_owner": self.owner,
-                    "path": self.path,
-                    "repo_name": self.repo_name
-                }
-        r = None
-        try:
-            r = requests.post(url, data=json.dumps(body), headers=headers)
-        except (ConnectionError, HTTPError, Timeout) as e:
-            logger.error('Connection exception : %s', e)
-        except:
-            logger.error('Unexpected Builder error: %s', sys.exc_info()[0])
+        repo_owner = self.repo_name.split("/")[0]
+        repo_name = self.repo_name.split("/")[1]
+        data = {
+            "github_token": self.oauth_token,
+            "git_hash": self.git_hash,
+            "repo_owner": repo_owner,
+            "path": self.path,
+            "repo_name": repo_name
+        }
 
-        if r is not None:
-            if (status.is_success(r.status_code) and 
-                    r.headers['Content-Type'] == 'application/json'):
-                building_status = r.json()['building']
-                if building_status:
-                    self.status = self.BUILDING
-                    self.save()
-                    return
-                else:
-                    logger.error("Negative response from Builder")
-            else:
-                logger.error('Builder responded without json')
-        else:
-            logger.error('Bad response from builder.')
-        self.status = self.FAILED
-        self.save()
+        # Push the data into the queue
+        # TODO: Add error handling somehow
+        q = Queue()
+        q.send_message(data)
```

Comment:
I do need to add some error handling. Will be easier to figure out how to handle errors once I've tested a bit.